rikai/public/js/custom.js

```diff
@@ -0,0 +1,568 @@
+//preloading for page
+$(window).on('load', function() { // makes sure the whole site is loaded 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
$(window).on("load", function() { // makes sure the whole site is loaded 
```